qa/tasks/cephfs/test_exports.py

```diff
@@ -25,6 +25,7 @@ def _wait_subtrees(self, status, rank, test):
     def test_export_pin(self):
         self.fs.set_allow_multimds(True)
         self.fs.set_max_mds(2)
+        self.fs.wait_for_daemons()
```

Comment:
wait_for_daemons() checks cluster health. I think cluster is health even when there is creating mds.

_setup_two_ranks() in test_strays.py use 
<pre>
        # See that we have two active MDSs
        self.wait_until_equal(lambda: len(self.fs.get_active_names()), 2, 30,
                              reject_fn=lambda v: v > 2 or v < 1)</pre>